client/my-sites/checkout/get-thank-you-page-url/index.ts

```diff
@@ -228,6 +232,13 @@ export default function getThankYouPageUrl( {
 		}
 
 		// siteless checkout
+		if ( connectAfterCheckout && adminUrl && fromSiteSlug ) {
+			debug( 'Redirecting to the site to initiate Jetpack connection' );
+			// TODO: Then after connection, transfer temporary site subscription to the target site.
+			const connectUrl = `${ adminUrl }/admin.php?page=jetpack&connect_url_redirect&from=my-jetpack&redirect_after_auth=${ adminUrl }/admin.php?page=my-jetpack#/add-license`;
```

Comment:
All the outputs of this function are covered by tests (see the top of this file for details). Could you add a test for this case?

Reply:
Yes of course, I should know better, my bad.  I have updated/modified this file many times and I know I need to add test coverage, and I was meaning to..., but then got to working on things and it slipped my mind. My apologies.  Yes, test coverage coming right up! 👍 😉